test/fixtures/projects/index.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs');
+const path = require('path');
+const android = require('./android');
+
+const dependencies = {
+  'react-native-vector-icons': {
+    'package.json': fs.readFileSync(path.join(__dirname, '../files/dependency.json')),
+    android: android.valid,
+  },
+  'rnpm-android-no-package': {
+    'package.json': fs.readFileSync(path.join(__dirname, '../files/dependency.json')),
+    android: android.noPackage,
+  },
+};
+
+const flat = {
+  android: android.valid,
+  node_modules: dependencies,
+};
+
+const nested = {
+  android: {
+    app: android.valid,
+  },
+  node_modules: dependencies,
+};
+
+module.exports = {
+  'testing/flat': flat,
+  'testing/nested': nested,
+  'testing/empty': {},
+};
```

Comment:
This is probably unnecessary and can be constructed on per `test case` basis instead of being the whole tree returned. But since we were using almost all of them in every test spec, I decided to leave it for now.